src/dev/impl/DevToys/ViewModels/Tools/Text/StringUtilities/StringUtilitiesToolViewModel.cs

```diff
@@ -626,6 +634,189 @@ private void ExecuteInverseCaseCommand()
 
         #endregion
 
+        #region AlphabetizeCommand
+
+        public IRelayCommand AlphabetizeCommand { get; }
+
+        private void ExecuteAlphabetizeCommand()
+        {
+            string? text = _textBackup;
+            if (text is null)
+            {
+                return;
+            }
+
+            var lines = text.Split('\r').ToList();
+            lines.Sort(StringComparer.CurrentCulture);
+            text = string.Join('\r', lines);
+
+            lock (_lockObject)
+            {
+                _forbidBackup = true;
+                Text = text;
+                _forbidBackup = false;
+            }
+        }
+
+        #endregion
+
+        #region ReverseAlphabetizeCommand
+
+        public IRelayCommand ReverseAlphabetizeCommand { get; }
+
+        private void ExecuteReverseAlphabetizeCommand()
+        {
+            string? text = _textBackup;
+            if (text is null)
+            {
+                return;
+            }
+
+            var lines = text.Split('\r').ToList();
+            lines.Sort(StringComparer.CurrentCulture);
```

Comment:
Why `ToList` and `Sort` instead of `OrderBy`?

Reply:
Good point, improved in 9c4d7228d8a6e92848b7f1277c343749ae964b33.